### ISSUE TITLE
Checklist ticking off implemented algorithms with references?

### ISSUE THREAD

**Issue by mohamed82008** [andreasnoack/LinearAlgebra.jl#26]
This is a great package! It would be nice to have a checklist with references to help contributors or generally curious people much like this issue https://github.com/JuliaMath/IterativeSolvers.jl/issues/1.

**Comment by andreasnoack** [comment]
Thanks. I might try to write something up as part of making this package ready for 0.7.

**Comment by chriscoey** [comment]
Agreed - I'd like to try using this package and maybe contribute, but without a roadmap/plan I can't tell whether it's worth the effort.

**Comment by mohamed82008** [comment]
Might be worth revisiting this to make this package more approachable for GSoC students.
